src/app/components_ngrx/group-types-panel/group-types-panel.component.html

```diff
@@ -13,7 +13,7 @@
         [class.dib]="sidePanelOpen"
         [class.hide]="!sidePanelOpen">
       {{ item.name }}</span>
-      <infotip [infotipText]="item.infotip"></infotip>
+      <infotip [uuid]="item.id"></infotip>
```

Comment:
The Infotip component should be pure, thus should only accept the string text, not a UUID so that we can use it anywhere else out of OSIO.